junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/InvocationInterceptor.java

```diff
@@ -207,6 +207,10 @@ default void interceptAfterAllMethod(Invocation<Void> invocation,
 		 */
 		T proceed() throws Throwable;
 
+		/**
+		 * Skip this invocation.
+		 */
+		void skip();
```

Comment:
Let's make this a default method with an empty implementation.

Reply:
This was my first intent then I was afraid that external extension should not be aware of this new skip() method if I provides a default implementation.

Reply:
The interface is not intended to be implemented by extensions so I think that should be safe.